argo-cd-bridges/gitlab/.helm/templates/imagestream.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    app: {{ include "launcher.fullname" . }}
+  name: {{ include "launcher.fullname" . }}
+spec:
+  lookupPolicy:
+    local: false
```

Comment:
Might need to change this to `true` based on my comment above. See https://docs.openshift.com/container-platform/3.11/dev_guide/managing_images.html#using-is-with-k8s for more info

Reply:
I hadn't seen that (`local: false` was just the default here). Thanks!